helm/loki/Chart.yaml

```diff
@@ -3,15 +3,15 @@ apiVersion: v2
 name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
-appVersion: 2.6.1
+appVersion: 2.7.0
 version: 0.5.3
 home: https://github.com/giantswarm/loki-app
 sources:
   - https://github.com/giantswarm/loki-app
 icon: https://s.giantswarm.io/app-icons/loki-stack/1/dark.svg
 dependencies:
   - name: loki
-    version: 3.2.1
+    version: 3.10.0
     repository: https://grafana.github.io/helm-charts
```

Comment:
Changing the source is for the next upgrade?

Reply:
Oh, forgot about that.
Could you elaborate?
You mean, just adding the current repo to the list of sources?

Reply:
No no, the fact that the https://github.com/grafana/helm-charts/tree/main/charts/loki is in maintenance mode and the chart moved to https://github.com/grafana/loki/tree/main/production/helm/loki. Maybe the chart repository is the same but the helm template was moved only, I'm not really sure considering https://grafana.github.io/helm-charts seems to still have 4.10.